### PR TITLE
Add Healthcheck route

### LINKED ISSUE #77: Add Healthcheck route
This is a route that connects to the database, connects to ldap, and returns a json blob stating whether or not it can connect.

This should allow us to more reasonably monitor on whether or not profiles is in working order. The route should not be behind auth so datadog can hit it

How I want this to work
```
GET /api/v1/healthcheck
```
That would return 200 only when both ldap and the database connect correctly, and a 5XX range response when they don't. Different 5XX code for ldap down, sql down and both down. Return the json no matter what the status code is.
```json
{
    "ldap": {
        "status": true,
        "server": "stone.csh.rit.edu"
    },
    "database": {
        "status": true,
        "server": "mysql.csh.rit.edu"
    }
}
```

### PR DESCRIPTION
**Pull request by adamhb123**
Adds a route that connects to and tests the database and ldap,
returning a json blob containing results. Resolves #77.

### CODE REVIEW

File: profiles/__init__.py [hunk 1]
```diff
@@ -192,9 +193,36 @@ def handle_internal_error(e):
...
+    }
+    try:
+        cshldap = csh_ldap.CSHLDAP(app.config['LDAP_BIND_DN'], app.config['LDAP_BIND_PASS'])
+        jsonout["ldap"]["server"] = cshldap.server_uri
+    except ldap.LDAPError:
+        jsonout["ldap"]["status"] = False
+        return_code = 512
+    try:
+        test_db = SQLAlchemy(app)
+        test_db.engine.connect()
```
Comment (mxmeinhold) [inline profiles/__init__.py:222]: Similar, is there a way to check this without reinitialising every time?

File: profiles/__init__.py [hunk 2]
```diff
@@ -1,11 +1,12 @@
 import os
-
```
Comment (mxmeinhold) [inline profiles/__init__.py:2]: This newline serves to separate python library imports from third party imports. It helps delineate import ordering visually.

Comment (adamhb123) [reply profiles/__init__.py]: Fixed

File: profiles/__init__.py (hunk 1 above)
Comment (adamhb123) [reply profiles/__init__.py:223]: Couldn't find a straightforward way to do this for the db. There may exist a solution but I don't see this as being a major issue.

File: profiles/__init__.py [hunk 3]
```diff
@@ -192,9 +193,36 @@ def handle_internal_error(e):
...
+            "status": True,
+            "server": None,
+        },
+        "database": {
+            "status": True,
+            "server": None,
+        }
+    }
+    try:
+        cshldap = csh_ldap.CSHLDAP(app.config['LDAP_BIND_DN'], app.config['LDAP_BIND_PASS'])
```
[the remark this answers is not in the document]
Comment (adamhb123) [reply profiles/__init__.py]: Fixed

File: profiles/__init__.py (hunk 1 above)
Comment (mxmeinhold) [reply profiles/__init__.py:223]: Alright, LGTM

**Review by mxmeinhold — APPROVED** [review]
LGTM! :ship: :it:

**Pull request merged**